project-base/app/src/Model/Order/OrderFacade.php

```diff
@@ -312,6 +312,9 @@ protected function fillOrderProducts(BaseOrder $order, BaseOrderPreview $orderPr
                 $product,
             );
 
+            $this->em->persist($orderItem);
+            $this->em->flush();
```

Comment:
Shouldn't be the fix also made in the `\Shopsys\FrameworkBundle\Model\Order\OrderFacade::fillOrderProducts`?